exercises/binary/src/example.c

```diff
@@ -0,0 +1,23 @@
+int len(char *str)
+{
+   int count = 0;
+   while (*str++) {
+      count++;
+   }
+   return count;
+}
+
+int convert(char *input)
+{
+   int input_len = len(input);
```

Comment:
I could be wrong but I thought the principle of not using provided functions referred to the conversion stated in the problem, not any incidental functions called in order to implement that solution? 
I.e. using functions from `string.h` etc would be OK, but if `string.h` provided a `strtobin()` function then using that would not be.